ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -5838,10 +5838,20 @@ public void dropConstraint(String dbName, String tableName, String constraintNam
     }
   }
 
-  public SQLAllTableConstraints getTableConstraints(String dbName, String tblName)
+  public SQLAllTableConstraints getTableConstraints(String dbName, String tblName, long tableId)
       throws HiveException, NoSuchObjectException {
     try {
-      return getMSC().getAllTableConstraints(new AllTableConstraintsRequest(dbName, tblName, getDefaultCatalog(conf)));
+
+      ValidWriteIdList validWriteIdList = null;
```

Comment:
Please move this logic from line 5845 - 5848 to a separate method as its getting repeated.

Reply:
Make sense. Actually there already exist a method created by you for same. I will replace this code with getValidWriteIdList(String dbName, String tableName).